### PR TITLE
instructions for aws customer to get arn by creating a policy and role

### PR DESCRIPTION
**Pull request by infinitewarp**
I just went through all the steps myself, and I wanted to get this documented somewhere that we could reference as we're defining the instructional needs of the UI.

FWIW, these docs **are not** intended to be considered final use for end users. This is effectively a first draft that I want to get posted so that we as developers and people defining the project have an understanding of what's needed of the customer in this process.

### CODE REVIEW

File: docs/aws-create-policy-and-role-for-arn.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                       "ec2:ModifyImageAttribute",
+                       "ec2:DescribeSnapshots"
+                   ],
+                   "Resource": "*"
+               }
+           ]
+       }
+
+#. Click "Review policy".
+#. Give the policy a distinct, memorable name such as ``policy-for-red-hat-cloud-meter``. Copy this name for reference because you will need it soon.
```
Comment (elyezer) [inline docs/aws-create-policy-and-role-for-arn.rst]: I would avoid red hat and cloud meter here, that said I would suggest to name to `policy-for-cloudigrade` instead.

File: docs/aws-create-policy-and-role-for-arn.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+#. Click "Create policy".
+
```
Comment (elyezer) [inline docs/aws-create-policy-and-role-for-arn.rst]: I think worth adding a note for each Action that snippet is allowing. Or point to AWS docs so user may review why those permissions are needed.

File: docs/aws-create-policy-and-role-for-arn.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+   #. log in to AWS console
+   #. search Services to go to IAM
+   #. click Roles in the left nav
+   #. click the "Create role" button
+   #. Click "Another AWS account"
+   #. Paste this number into the Account ID field: ``372779871274``
+
+#. Click "Next: Permissions".
+#. Enter your new policy's name in the search box (e.g. ``policy-for-red-hat-cloud-meter``)
```
Comment (elyezer) [inline docs/aws-create-policy-and-role-for-arn.rst]: Again I would recommend naming to `policy-for-cloudigrade`

File: docs/aws-create-policy-and-role-for-arn.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+#. Check the box for the policy.
+#. Click "Next: Review".
+#. Give the role a distinct, memorable name such as ``role-for-red-hat-cloud-meter``.
```
Comment (elyezer) [inline docs/aws-create-policy-and-role-for-arn.rst]: I would recommend naming to `role-for-cloudigrade`

File: docs/aws-create-policy-and-role-for-arn.rst [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+#. Click "Create role".
+#. From the Roles list page, click the role name you just created.
+#. Copy the generated value for "Role ARN".
```
Comment (elyezer) [inline docs/aws-create-policy-and-role-for-arn.rst]: I think a note explaining why this role is needed would be welcome. Better to be transparent, right?

**Comment by infinitewarp** [comment]
Thanks for all the suggestions, @elyezer! I've taken the time here to clean up our required permissions and explain why we need each one.

To anyone else interested in this PR, please keep in mind that this is **not** intended to be final user-facing copy. This is effectively a first draft that I want to get posted so that we as developers and people defining the project have an understanding of what's needed of the customer in this process.

**Comment by elyezer** [comment]
@infinitewarp thanks for the updates. That helps everyone trying to learn more about why those are needed, also it gives some clues about test cases we can explore, for example, what if a given permission is missing.

**Pull request merged**